openstudio-standards/lib/openstudio-standards/prototypes/Prototype.utilities.rb

```diff
@@ -145,14 +145,8 @@ def hspf_to_cop_heating_no_fan(hspf)
 # @param [Double] COP
 # @return [Double] Seasonal Energy Efficiency Ratio
 def cop_to_seer(cop)
-  seer = nil
-
-  # First convert COP to EER
-  eer = cop_to_eer(cop)
-
-  # Next convert from EER to SEER
-  delta = 1.1088**2 - 4.0 * 0.0182 * eer
-  seer = (-delta**0.5 + 1.1088) / (2.0 * 0.0182)
+  delta = 0.3796**2 - 4.0 * 0.0076 * cop
```

Comment:
Do you have a citation for this change?  Right now, the comments in the method point to this document by name:

https://www.energycodes.gov/sites/default/files/documents/BECP_Energy_Cost_Savings_STD2010_May2011_v00.pdf